fsspec/implementations/tests/test_tar.py

```diff
@@ -42,7 +42,7 @@ def test_info():
             assert lhs == expected
 
         # Iterate over all files.
-        for f, v in archive_data.items():
+        for f in archive_data.keys():
```

Comment:
```suggestion
        for f in archive_data:
```

Reply:
@martindurant If you prefer that I have a separate rule that does this automatically repo wide.

Reply:
Using `.keys()` I thought had generally disappeared from python since 3.0; at least, if the object in question is really a dict.